packages/nvidia_nat_core/src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -352,1300 +319,74 @@ async def configure(self, app: FastAPI, builder: WorkflowBuilder):
-                                          "example": {
-                                              "status": "healthy"
-                                          }
-                                      }
-                                  }
-                              }
-                          })
-
-        logger.info("Added health check endpoint at /health")
+                        session_manager: SessionManager,
+                        *,
+                        enable_interactive: bool = False,
```

Ruff (0.15.0)

[warning] Unused method argument: `enable_interactive`

(ARG002)